docs/gaia/launch/blog-1-sp.md

```diff
@@ -0,0 +1,181 @@
+meses de 2019, el software está [completo](https://github.com/cosmos/cosmos-sdk/releases). El lanzamiento del Hub del Cosmos se acerca. 
+¿Qué significa esto para los poseedores de átomos?
+
+Si eres poseedor de Atoms, podrás delegar Atoms en los validadores de la red principal y votar sobre las propuestas de 
+gobernanza. De hecho, el éxito futuro de la red depende de que lo haga de forma responsable. Sin embargo, aún no podrás 
+transferir Atoms. Las transferencias se desactivarán a nivel de protocolo hasta que se ejecute un hard-fork para habilitarlas.
+
+Los poseedores de Atoms deben seguir cuidadosamente las directrices para poder delegar Atoms de forma segura. Por favor, 
+lea primero toda la guía para familiarizarse antes de hacer nada: [Guía CLI](https://github.com/cosmos/cosmos-sdk/blob/develop/docs/gaia/delegator-guide-cli.md)
+
+El proceso descrito en la guía es actualmente la única forma verificada y segura de delegar Atoms en el lanzamiento. Esto se debe a que 
+la herramienta gaiacli utilizada en la guía es el único software de cartera que está siendo sometido a auditorías de seguridad 
+por parte de terceros en este momento. Ningún otro proveedor de carteras ha iniciado aún las auditorías de seguridad.
+
+Recuerde que delegar Atoms implica un riesgo significativo. Una vez delegados a un validador, los Atoms son fijados por un 
+período de tiempo durante el cual no pueden ser recuperados. Si el validador se comporta mal durante este tiempo, algunos o 
+todos los Atoms delegados pueden ser destruidos. Es su responsabilidad realizar la debida diligencia con los validadores antes 
+de delegar!
+
+The Cosmos Hub es un software altamente experimental. En estos primeros días, podemos esperar tener problemas, actualizaciones 
+y errores. Las herramientas existentes requieren conocimientos técnicos avanzados e implican riesgos que están fuera del control 
+de la Fundación Interchain y/o del equipo de Tendermint (ver también la sección de riesgos en [Interchain Cosmos Contribution Terms](https://github.com/cosmos/cosmos/blob/master/fundraiser/Interchain%20Cosmos%20Contribution%20Terms%20-%20FINAL.pdf)). 
+Cualquier uso de este software de código abierto [con licencia Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) se hace bajo su propio riesgo y sobre una 
+base "TAL CUAL" sin garantías ni condiciones de ningún tipo, y toda responsabilidad de la Fundación Interchain y/o del equipo 
+de Tendermint por daños y perjuicios. que surjan en relación con el software. Por favor, haz ejercicio extremo de precaución!
+
+Si buscas más información sobre la delegación y quieres hablar con la gente que está desarrollando Cosmos, únete a la reunión 
+virtual del 14 de febrero, en la que se te explicarán paso a paso las instrucciones para delegar Atoms en el lanzamiento.
+
+Regístrese aquí: [gotowebinar.com/register/](https://register.gotowebinar.com/register/5028753165739687691)
```

Comment:
Same as above, (I think) it's more correct to talk to them as "usted" instead of "tú".
But if you think it's more correct like you say I can change it.